input/vlct/run_HD_linear_wave_test.py

```diff
@@ -0,0 +1,138 @@
+#!/bin/python
+
+# runs VLCT HD Linear Wave Tests
+# - This should probably be transitioned to testing framework once it's created
+# - This script expects to be called from the root level of the repository
+#   OR at the same level where its defined
+#
+# These tests draw loose inspiration from Linear Wave tests used by Athena++.
+# Specifically, this script:
+#   1.) Checks if that L1-norm of the error is appropriate for the inclined
```

Comment:
As discussed during the all a test for the L_inf norm would also be great, but this may also be better suited for the test framework overhaul.
Feel free to resolve this comment and open an issue to keep track of this.

Reply:
As I recall, that conversation was in the context of checking flux corrections had more to do with testing flux corrections.

But, I do agree that having L_inf norm checks would probably be good (although, I imagine it would be easier to implement them with axis aligned waves). I'll be sure to add the running list of testing improvements (that I'll create an issue for)